### DISCUSSION TITLE
Welcome to Akira Discussions!

### DISCUSSION THREAD

**Discussion by Alecaddd** [akiraux/Akira#462] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LucileDT** [comment]
Hi! I'm a web developer, mainly using PHP and Symfony to work. I'm a French native speaker and I'm fluent in English. I'm mostly here because I'm interested in Akira UX, as I need a tool to make mockups on a daily basis and all the current usable options are way too pricy for me. I can help with translations and I can alpha/beta test Akira UX and make complete issues describing bugs encountered/features which could be great. Didn't do it until now because I wait to have an easy way to install Akira :computer:

**Comment by omegaui** [comment]
I am so excited about Akira.
It's amazing.
I can't contribute cause I don't know Vala (Only know Java and Python)
But I must say it's amazing.
I love it.
Even in it's alpha state.
💖💖💖💖💖💖💖💖

**Reply by ghost** [reply]
Watch Alecaddd Vala Series (https://www.youtube.com/watch?v=0R8gzwiR9sI&t=355s)

**Comment by Golotvinov** [comment]
An open-source UI / UX tool that is comparable to the big guys (XD, Figma, Sketch)? It could be possible. Super exciting project. I'm not familiar with the Vala programming language, but I'm willing to help you guys with all I can. (currently working as a graphic and UI / UX and web designer.

**Comment by Alecaddd** [comment]
Quickly passing by to send you all a big thank you hug for your support and patience.
But a particular shout out to @LucileDT for her constant support.
Even tho you're not a developer and you can't contribute to the project from that point of view, it's always a pleasure seeing your thumbs up and emoji feedback on the various PRs.
It gives me a boost every time :D

**Comment by Suzie97** [comment]
Hi, I'm a college student studying UX/UI design. Akira interests me because I love linux and open source. I believe that Akira will be a professional tool for designers and I can use it as my daily driver once I'm a professional.

I've been learning Vala and Gtk recently and I'm working on a cute clock widget for elementary OS so that I can practice Vala and Gtk which is gonna help me in my future project ideas. 

I've got a lot of ideas that I would want to bring to life, some of them are really hard and I don't know if I'll be able to fulfill them or not, like making a pro level DAW for Linux. But I know I'll have the support of the wonderful Linux and open source community, so maybe it can be a reality.

PS I'm not very good in english.

**Reply by ghost** [reply]
Hi @Suzie97 you can join our matrix server https://matrix.to/#/!kpfeTRbpocQrOFCFnJ:matrix.org

**Reply by ghost** [reply]
Here we communicate and if you need help we will answer your questions
